src/pbt/prophecy_build_tool.py

```diff
@@ -729,3 +732,22 @@ def _verify_project(self):
     def _error(cls, message: str):
         print("[bold red]ERROR[/bold red]:", message)
         sys.exit(1)
+
+    def _setJarsNeededForUT(self, build_jars):
+        fallBackSparkPackages = [
+            "com.typesafe.play:play-functional_2.12:2.6.13",
+            "com.typesafe.play:play-json_2.12:2.6.13",
+            "com.typesafe.akka:akka-http-core_2.12:10.1.6",
+            "joda-time:joda-time:2.12.5",
+            "com.crealytics:spark-excel_2.12:3.2.2_0.18.0",
+            # "net.sf.py4j:py4j:0.10.9.2"
+            "io.prophecy:prophecy-libs_2.12:6.3.0-3.1.2"
+        ]
+        import random
+        uniqueKey = random.random()
+        jars_unique_key: str = f"driver_library_path_{uniqueKey}"
+        os.environ["JARS_GIVEN_BY_USER"] = build_jars if build_jars and (build_jars != "") else ",".join(fallBackSparkPackages)
```

Comment:
Might be better to give a env name to reflect the spark conf being set.

suggestion : SPARK_JARS_CONFIG